### PR TITLE
Updating docs to use 'g' suffix instead of 'm' for disk sizes

### PR DESCRIPTION
**Pull request by tpill90**
- Updated docs to use 'g' suffix instead of 'm' suffix for CACHE_DISK_SIZE
- Documented how to run the docs locally on Windows

### CODE REVIEW

**Comment by astrolox** [comment]
@VibroAxe can you spare a moment to let me know if this looks good to you?

File: README.md [hunk 1]
```diff
@@ -36,7 +36,13 @@ This repo contains the sourcecode and ancillary information for Lancache.net and
 ## Building the site
 
 ### Jekyll & Github Pages
-This site is built using Jekyll and hosted on Github pages. To build at home for testing see [jekyllrb.com](https://jekyllrb.com/docs/)
+This site is built using Jekyll and hosted on Github pages. To build at home for testing :
+
+- Install Ruby according to the Jekyll docs [Jekyll Windows Install](https://jekyllrb.com/docs/installation/windows/)
```
Comment (VibroAxe) [inline README.md:41]: Why would we link directly to the windows instructions here? The general install root gives links to each OS which is why we chose it. Please revert this change

**Review by VibroAxe — CHANGES_REQUESTED** [review]
Couple of comments and a change please

File: Gemfile [hunk 2]
```diff
@@ -14,3 +14,9 @@ source "https://rubygems.org"
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
 gem "github-pages", group: :jekyll_plugins
+
+gem "webrick", "~> 1.7"
```
[the remark this answers is not in the document]
Comment (tpill90) [reply Gemfile:18]: From Googling around it looks like Ruby used to have `webrick` included, but as of 3.0 they are no longer shipping it with Ruby.  `webrick` appears to be needed by Jekyll, the only odd thing is that it only needs to be installed for Window's version of Ruby.  I've tested Jekyll on Linux as well and it seems to not be required there.

I've updated the gemfile to conditionally install these gems only for Windows

File: Gemfile.lock
[the remark this answers is not in the document]
Comment (tpill90) [reply Gemfile.lock:1]: It looks like according to bundler the answer is yes.  From their site https://bundler.io/guides/bundler_sharing.html : 

> When your co-developers (or you on another machine) check out your code, it will come with the exact versions of all the third-party code your application used on the machine that you last developed on (in the Gemfile.lock). When they run bundle install, bundler will find the Gemfile.lock and skip the dependency resolution step. Instead, it will install all of the same gems that you used on the original machine.

File: README.md (hunk 1 above)
Comment (tpill90) [reply README.md:41]: My intention here was to try to reduce the friction required to get setup to be as low as possible, in the hopes that it will help encourage the community to contribute changes as much as possible.

I had no prior experience with Ruby or Jekyll, so I kept notes of the steps required to run Jekyll locally, and consolidated the steps onto the readme here.  The commands required (such as `bundle install`) are in their documentation, however I wanted to make no assumptions of people's experience level, and instead consolidate these steps front and center on this project's readme.

Regardless, I'll get this reverted and committed.

**Comment by tpill90** [comment]
@VibroAxe I've made the requested changes and pushed up a new commit.  Let me know if the changes look good, and I can squash the two commits if you'd like prior to merging.

**Review by VibroAxe — APPROVED** [review]
Lgtm
